Mathlib/Topology/PartialHomeomorph.lean

```diff
@@ -420,13 +420,13 @@ theorem eventually_nhds {x : α} (p : β → Prop) (hx : x ∈ e.source) :
 theorem eventually_nhds' {x : α} (p : α → Prop) (hx : x ∈ e.source) :
     (∀ᶠ y in 𝓝 (e x), p (e.symm y)) ↔ ∀ᶠ x in 𝓝 x, p x := by
   rw [e.eventually_nhds _ hx]
-  refine' eventually_congr ((e.eventually_left_inverse hx).mono fun y hy => _)
```

Comment:
As far as I can see, these changes are unrelated to this PR. Can you split them into a different PR?
(Making the change to a line you're modifying anyway is fine (and should perhaps be mentioned in the commit message; I think other lines should be separate.)